config/webpack.config.prod.js

```diff
@@ -26,6 +26,7 @@ var nodeModulesPath = path.join(__dirname, '..', 'node_modules');
 var indexHtmlPath = path.resolve(__dirname, relativePath, 'index.html');
 var faviconPath = path.resolve(__dirname, relativePath, 'favicon.ico');
 var buildPath = path.join(__dirname, isInNodeModules ? '../../..' : '..', 'build');
+var publicPath = require('./package.json').homepage || '/';
```

Comment:
I can do that, but then I run into an error that there is no `package.json` in the `template` folder. I can include a sample `package.json` in that folder and then the tests will successfully run - are you ok with that @gaearon ?